Content.Server/Physics/Controllers/MoverController.cs

```diff
@@ -478,19 +512,27 @@ private void HandleShuttleMovement(float frameTime)
                         totalForce += impulse;
                     }
 
-                    totalForce = shuttleNorthAngle.RotateVec(totalForce);
-
                     var forceMul = frameTime * body.InvMass;
-                    var maxVelocity = (ShuttleComponent.MaxLinearVelocity - body.LinearVelocity.Length()) / forceMul;
 
-                    if (maxVelocity != 0f)
-                    {
-                        // Don't overshoot
-                        if (totalForce.Length() > maxVelocity)
-                            totalForce = totalForce.Normalized() * maxVelocity;
+                    var localVel = (-shuttleNorthAngle).RotateVec(body.LinearVelocity);
+                    var maxVelocity = ObtainMaxVel(localVel, shuttle); // max for current travel dir
+                    var maxWishVelocity = ObtainMaxVel(totalForce, shuttle);
```

Comment:
Why is a force unit being passed into a function that works on velocity?

Reply:
At the limit, any vector in that direction applied as force will result in the shuttle reaching that velocity. The exact magnitude of the vector isn't important, which is why I plug both velocity and force into these functions unopposed.